make-money/break-moment.md

```diff
@@ -0,0 +1,27 @@
+# Break Moment
+
+Breaks provide custom brand messages during natural pauses in application use.  Since users expect pauses at certain points in your application, displaying ads at this time will not take away from the user experience. There are a variety of ads that can be displayed during a break moments including; rich media,video, and app download ads.
```

Comment:
Couple of typos in the sentence - fix as:
"... during a break moment including rich media, video, and app download ads."